docs/DafnyRef/UserGuide.md

```diff
@@ -171,6 +166,10 @@ A few options are not part of a command. In these cases any single-hyphen spelli
 
 The `dafny resolve` command checks the command-line and then parses and typechecks the given files and any included files.
 
+Note that a complete program must be presented to the dafny tool, either by listing all files on the command-line, or by using `include` directives,
+of by some combination. An incomplete program will likely result in errors because of unresolved names. A program need not include all implementations
+of all functions in order to verify parts of it, but will need all implementations in order to compile a working executable.
```

Comment:
How is it possible to provide a declaration à la c-header and an implementation on anther file? I don't see how.

Reply:
There is no analog of c-headers of course, but one can omit implementations of functions for verification, but must have them for compilation.

Reply:
I was wondering the same and this answer was helpful, so maybe it could be clearer?

Reply:
OK. I'll clarify in a future PR (since this one is merged) keeping in mind programmers coming from C.